source/Application/Component/Widget/ArticleDetails.php

```diff
@@ -646,9 +631,7 @@ public function showZoomPics()
      */
     public function getZoomPics()
     {
-        $aPicGallery = $this->getPictureGallery();
```

Comment:
This code gives advantage in debugging case. You can easily check whats the results in the variable. Does it really worth to remove the variables?